project-reports/2023/2023-Q2-Hyperledger-Cello.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: 2023 Q2 Hyperledger Cello
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+V1.0.0 version has been released.
```

Comment:
It doesn't seem like this has happened yet and it's contradicted by the following statement about working towards that very release. Please, clarify.

Reply:
With help from @tkuhrt and @ryjones I can now see that 1.0 really is out. So it's probably the next section that needs fixing. Also not that the CHANGELOG.md file is missing info on 1.0.